src/lgtm/scripts/lib/download/_.sh

```
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+DIR="$(dirname "${BASH_SOURCE[0]}")"
+
+source "$DIR/download.sh"
```

Shellcheck (0.10.0)

[info] Not following: ./download.sh was not specified as input (see shellcheck -x).

(SC1091)